.github/workflows/build-release-artifacts.yml

```diff
@@ -0,0 +1,105 @@
+name: Build Release Artifacts
+
+on: 
+  workflow_dispatch:
+
+jobs:
+  release-artifacts:
+    runs-on: ubuntu-latest
+      
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up environment
+      uses: ./.github/actions/setup-rel-env
+    
+    - name: Install krew
+      run: | 
+        make krew
+        echo "${KREW_ROOT:-$HOME/.krew}/bin" >> $GITHUB_PATH
+      
+    - name: Build CRD yaml
+      run: make generate manifests
+    
+    - name: Lint the charts
+      run: make lint
+
+    - name: Build vdb-gen
+      run: | 
+        mkdir -p bin
+        make vdb-gen
+        pwd
+        ls -lhrt bin/
+
+    - name: Build RBAC yaml
+      run: |
+        mkdir -p config/samples/rbac
+        make create-default-rbac
+        ls -lhrt config/samples/rbac/
+
+    - name: Build Bundle
+      run: | 
+        make bundle
```

Comment:
I think the image in the bundle for verticadb-operator won't be correct.  It will likely be `verticadb-operator:latest`.  We need to use the official image in dockerhub (i.e. `vertica/verticadb-operator:1.3.1`).  The tag comes from the operator version.

Currently, the image is defined in the Makefile like this:
```
OPERATOR_IMG ?= verticadb-operator:$(TAG)
```

Where `$(TAG)` is either `kind` or `latest`.

I think we should fix this by updating the default image/tag in the Makefile.  We don't need to pick the tag between kind/latest, lets just use the version.  We can add a new `$(REPO)` variable that allows you to set the repository.  This can apply to all of the `*_IMG` variables.

Then we use the correct image with this command:
```
REPO=vertica/ make bundle
```

Reply:
Lets default to an empty string.